[esx_addons]/esx_basicneeds/client/main.lua

```diff
@@ -114,16 +91,16 @@ AddEventHandler('esx_basicneeds:onDrink', function(prop_name)
 			local boneIndex = GetPedBoneIndex(playerPed, 18905)
 			AttachEntityToEntity(prop, playerPed, boneIndex, 0.12, 0.028, 0.001, 10.0, 175.0, 0.0, true, true, false, true, 1, true)
 
-			ESX.Streaming.RequestAnimDict('mp_player_intdrink', function()
-				TaskPlayAnim(playerPed, 'mp_player_intdrink', 'loop_bottle', 1.0, -1.0, 2000, 0, 1, true, true, true)
-				RemoveAnimDict('mp_player_intdrink')
+			print(json.encode(anim))
```

Comment:
remove plz

Reply:
> remove plz

I did remove it. Thank you for telling me to!